### PR TITLE
Add article about next steps after 1st published lesson

### PR DESCRIPTION
**Pull request by zacjones93**

### CODE REVIEW

**Comment by vojtaholik** [comment]
it was because of the apostrophe in `title`, should be `’` instead of `'` :)

**Comment by zacjones93** [comment]
ah, I just caught that. You don't need the quotes?

**Comment by vojtaholik** [comment]
it works without them, too 👍

**Comment by zacjones93** [comment]
Good to know!

File: content/instructor/communication/next-steps/index.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+date: 2019-03-21
+title: You’ve published your first lesson, now what?
```
Comment (joelhooks) [inline content/instructor/communication/next-steps/index.mdx]:
```suggestion
title: Congrats on publishing your first lesson! Now what?
```

File: content/instructor/communication/next-steps/index.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+description: 'Now that you have completed the full process of lesson creation, lets really nail it and get more practice in.'
+categories: ['instructor', 'communication']
+published: false
+---
+
+
+The first 30-second demo started with you uploading a clip to slack. Those days are past us now. Feel free to upload directly to egghead (to your hearts desire!) for the following lessons.
```
Comment (joelhooks) [inline content/instructor/communication/next-steps/index.mdx]:
```suggestion
Taking the first step is the hardest, but now you are on your way. Congrats on publishing your first lesson! We highly recommend jumping right into your next lesson as soon as possible to keep the momentum going.
```

File: content/instructor/communication/next-steps/index.mdx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+The process will largely be the same as the first lesson you produced. When you upload a lesson, you can link to it in your personal instructor chat and it will be reviewed and iterated on with a coach.
```
Comment (joelhooks) [inline content/instructor/communication/next-steps/index.mdx]:
```suggestion
From now on feel free to [upload your new lesson videos directly to egghead](https://egghead.io/lessons/new).

When you've uploaded a new lesson, let your coaches know in your Slack channel or send an email. Whatever works best for you.

If you're struggling to find something to teach, we can help you! You are not alone, and we are happy to help.
```

File: content/instructor/communication/next-steps/index.mdx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+You'll notice each lesson will get a little easier to record and publishing will come a little faster.
```
Comment (joelhooks) [inline content/instructor/communication/next-steps/index.mdx]:
```suggestion
Creating screencasts is a learned skill that requires practice. With each lesson you create, you should notice that it gets a little easier. If you're feeling stuck or frustrated, your coaches will help. This is hard to do, especially when you are just getting started and you have access to all the support we can offer.
```

File: content/instructor/communication/next-steps/index.mdx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+ 
+Thats progress. 🙌
```
Comment (joelhooks) [inline content/instructor/communication/next-steps/index.mdx]:
```suggestion
We are here to help you shine. 🙌
```

File: content/instructor/communication/next-steps/index.mdx [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+After 12 published lessons, you'll have the ability to self approve/publish your own lessons. We'll always be here for you if you'd like a review on a particular difficult lesson but at this point, we trust your judgement on producting lessons.
```
Comment (joelhooks) [inline content/instructor/communication/next-steps/index.mdx]:
```suggestion
For now, we want to focus on publishing single one-off lessons, but once you've published a handful of those, we can start to discuss your first course.

After you've published 12 lessons, you'll gain the ability to self approve/publish your own lessons. This is a big step, and by that point, you will feel comfortable and confident, but we are here to help no matter how many lessons you've published.
```

Comment (zacjones93) [reply content/instructor/communication/next-steps/index.mdx]: github wouldn't let me commit this suggestion, did it manually

**Pull request merged**